### PR TITLE
Adding technical details from the unobfuscated stage3 sample

### PR DESCRIPTION
**Pull request by Col-E**
Added some details that we uncovered from the unobfuscated stage3, and obfuscated stage3b jars

### CODE REVIEW

File: docs/tech.md [hunk 1]
```diff
@@ -142,11 +143,31 @@ There is also evidence of code attempting to do the following:
...
 
 Jars are heuristically detected as Minecraft mods or plugins as follows:
-* Forge (dev/neko/e/e/e/A): The malware attempts to locate the `@Mod` annotation, which is required in every mod
-* Bukkit (dev/neko/e/e/e/C): The malware checks if a class extends Bukkit's `JavaPlugin` class
-* Fabric/Quilt (dev/neko/e/e/e/i): The malware checks if a class implements `ModInitializer`
-* Bungee (dev/neko/e/e/e/l): The malware checks if a class extends Bungee's `Plugin` class
-* Vanilla (dev/neko/e/e/e/c): The malware checks if the main client class `net.minecraft.client.main.Main` exists
+* Forge (`dev/neko/e/e/e/A`): The malware attempts to locate the `@Mod` annotation, which is required in every mod
+* Bukkit (`dev/neko/e/e/e/C`): The malware checks if a class extends Bukkit's `JavaPlugin` class
+* Fabric/Quilt (`dev/neko/e/e/e/i`): The malware checks if a class implements `ModInitializer`
```
Comment (SilverAndro) [inline docs/tech.md:148]: how is this check performed? quilt has a different package for its ModInitializer

Comment (Col-E) [reply docs/tech.md:148]: The bullet points for the unobf version further below explain more in detail

Comment (SilverAndro) [reply docs/tech.md:148]: Yeah this wont affect quilt then, unless someone was relying on qfapi and ignored the depreciation notice

**Comment by williewillus** [comment]
amazing, thanks

File: docs/tech.md [hunk 2]
```diff
@@ -145,24 +146,107 @@ There is also evidence of code attempting to do the following:
...
 
-When the second C&C server was stood up, a deobfuscated version of Stage 3 was
-accidentally served for around 40 minutes.
+This validates the suspected behavior/evidence from the analysis done on the prior obfuscated `client.jar` sample.
+
+### Replication
+
+Replication is handled through automatic processing of classes in jar files across the entire filesystem on the local machine. Any jar file that contains classes meeting certain critera is subject for infection. The process of scanning the local file system and injecting malicious code can be found here: [`dev/neko/nekoclient/Client.start(InetSocketAddress, byte[])`](https://github.com/clrxbl/NekoClient/blob/main/dev/neko/nekoclient/Client.java#L273)
+
+The critera that the process looks for can be found here: [`dev/neko/nekoinjector/template/impl`](https://github.com/clrxbl/NekoClient/tree/main/dev/neko/nekoinjector/template/impl)
```
[the remark this answers is not in the document]
Comment (Col-E) [reply docs/tech.md:165]: Oops, feel free to open a PR to add this note

**Pull request merged**